themes/rose-pine-dawn.json

```diff
@@ -3,12 +3,12 @@
     "\"UTF.8\"": true
   },
   "$schema": "https://zed.dev/schema/themes/v0.1.0.json",
-  "name": "Rosé Pine",
+  "name": "Rosé Pine Dawn",
   "author": "Kainoa Kanter <kainoa@t1c.dev>",
   "themes": [
     {
       "appearance": "dark",
```

Comment:
Unrelated but should this be light instead of dark?

Reply:
@mvllow Would make sense, unsure what it would accomplish. In Zed, theme switching based on system appearance seems to only go by name. Perhaps some categorising or grouping, guess it wouldn't hurt?